src/Compilers/Core/CommandLine/BuildProtocol.cs

```diff
@@ -60,12 +60,21 @@ internal class BuildRequest
             }
         }
 
+#if USES_ANNOTATIONS
```

Comment:
❓ If this is in a branch anyway, why have the preprocessor like this? It makes it hard to see the true impact 😦

Reply:
Correct. I'm setting LangVersion in MSBuildTask first, then other projects will follow.